openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/certificate_estimation/HVAC2_ESC_variables.py

```diff
@@ -43,27 +44,42 @@ class HVAC2_equivalent_heating_hours_input(Variable):
     value_type = float
     entity = Building
     definition_period = ETERNITY 
+    
+    metadata = {
+        "variable-type": "inter-interesting"
+    }
 
 
 class HVAC2_equivalent_cooling_hours_input(Variable):
     reference = 'table_F4.1'
     value_type = float
     entity = Building
     definition_period = ETERNITY 
+    metadata = {
+        "variable-type": "inter-interesting"
+    }
+
 
 
 class HVAC2_baseline_ACOP_input(Variable):
     reference = 'table_F4.3'
     value_type = float
     entity = Building
     definition_period = ETERNITY 
+    metadata = {
+        "variable-type": "inter-interesting"
+    }
+
 
 
 class HVAC2_baseline_AEER_input(Variable):
     reference = 'table_F4.3'
     value_type = float
     entity = Building
     definition_period = ETERNITY 
+    metadata = {
+        "variable-type": "inter-interesting"
```

Comment:
for all intermediary inputs for which we don't want to show user input, we are using this variable type in the codebase